### DISCUSSION TITLE
Support for Safari 14

### DISCUSSION THREAD

**Discussion by GaborTheFisher** [Cookie-AutoDelete/Cookie-AutoDelete#1003] [Ideas / Feature Requests] [answered]
This is a great tool that I'm currently using on all my browsers (Firefox, Brave) except Safari. Would love to see Safari support being added if this is technically possible.

**Comment by kennethtran93** [comment, marked as answer]
Duplicate of #964.

A more detailed reason is posted there, but long story short:  while technically possible (albeit with some existing features that will need to be disabled), the major drawback at this time is the financial constraint.

**Reply by GaborTheFisher** [reply]
Understand, thank you for clarifying. And a paid version of the extension for apple's platform is not something you'd like to consider?

**Reply by kennethtran93** [reply]
Why have one browser version be paid webextension when all the other published versions are free? 

Majority of the other addon publishing platforms are free as well:
Chrome (after a $5 one-time developer fee),
Firefox,
Edge.

Not to mention the slim percentage of users in Safari (last checked online anyway)

If we do put a price on the webextension, it would only be a small one-time item, vs the yearly membership develop fee.
